src/Cpp/1-getting-started/1-3-3-SettingUp3DRendering/ShaderCollection.hpp

```diff
@@ -17,11 +17,17 @@ struct ShaderCollectionDescriptor
 class ShaderCollection
 {
 public:
+
     static ShaderCollection CreateShaderCollection(
         const ShaderCollectionDescriptor& settings, ID3D11Device* device);
+
     static size_t GetLayoutByteSize(VertexType vertexType);
+
     void Set(ID3D11DeviceContext* context);
```

Comment:
Hmm where would it make sense to expose this function, when you pass the device in the class factory already? (deferred contexts are also kind of a not worthy endeavour in d3d11)